common/api.go

```diff
@@ -20,8 +20,9 @@ func NewInflux(influx config.Influx) (*Influx, error) {
 }
 
 // Query runs a query against this backend.
-func (d *Influx) Query(queryStr, epoch string) (*client.Response, error) {
-	return d.handle.Query(queryStr, d.data.Database, epoch)
+func (d *Influx) Query(
+	logger *log.Logger, query *influxql.Query, epoch string) (*client.Response, error) {
```

Comment:
Can you switch to the Logger interface type from github.com/Symantec/Dominator/lib/log ?

Reply:
Yep, I got a story for switching to lib/log.